wandb/data_types.py

```diff
@@ -112,7 +112,12 @@ def _json_helper(val, artifact):
         return res
 
     if hasattr(val, "tolist"):
-        return _json_helper(val.tolist(), artifact)
+        py_val = val.tolist()
+        if val.__class__.__name__ == "datetime64" and isinstance(py_val, int):
+            # when numpy datetime64 .tolist() returns an int, it is nanoseconds.
```

Comment:
Curious fact I just learned: "The data type is called “datetime64”, so named because “datetime” is already taken by the datetime library included in Python."